lib/matplotlib/_mathtext.py

```diff
@@ -767,6 +767,23 @@ class FontConstantsBase:
     # for slanted nuclei
     delta_slanted = 0.2
 
+    # Percentage of x-height the numerator is shifted up in display style.
+    num1 = 1.4
+
+    # Percentage of x-height the numerator is shifted up in text style if there
+    # is a fraction line.
+    num2 = 1.5
```

Comment:
Are these names used in TeX? If not, can we have slightly more descriptive names? e.g. `num_disp`, `num_text_frac`, `num_text_nofrac`

Reply:
Yes, these are the names used in TeX. We already have `subdrop`, `sup1`, `sub1` and `sub2` (which are also the names of TeX font constants) in `FontConstantsBase` so I ran with it and used TeX's names (`num1`, `num2`, `num3`, `denom1` and `denom2`) for the new constants.